modes/tmtv/src/index.js

```diff
@@ -131,6 +133,43 @@ function modeFactory({ modeConfiguration }) {
         'RectangleROIStartEndThreshold',
         'fusionPTColormap',
       ]);
+
+      // For the hanging protocol we need to decide on the window level
+      // based on whether the SUV is corrected or not, hence we can't hard
+      // code the window level in the hanging protocol but we add a custom
+      // attribute to the hanging protocol that will be used to get the
+      // window level based on the metadata
+      hangingProtocolService.addCustomAttribute(
```

Comment:
It feels like this should come as part of an extension rather than being in the mode directly.  I'm ok leaving it here for now, but eventually this sort of logic should get incorporated into an extension object.

Reply:
As talked we will do this later